src/libraries/Microsoft.Extensions.Configuration.Binder/src/buildTransitive/Microsoft.Extensions.Configuration.Binder.targets

```diff
@@ -8,6 +8,14 @@
         </ItemGroup>
     </Target>
 
+    <Target Name="_Microsoft_Extensions_Configuration_Binder_EnableInterceptors"
```

Comment:
It looks like we're not removing the old feature flag in this change set. From looking at [this commit](https://github.com/dotnet/runtime/blob/release/8.0-rc1/src/libraries/Microsoft.Extensions.Configuration.Binder/src/buildTransitive/Microsoft.Extensions.Configuration.Binder.targets) it doesn't look like it was ever included.

I assume this means that behavior for RC1 + NuPkg will mean that users have to manually set the flag themselves?

Reply:
That's right. For RC-1 the question of enabling interceptors implicitly (along with the other dimension of behavior diff between web SDK vs not) was still very open. I decided not to pursue adding the implicit opt-in here given the concerns around the global opt-in.

I think the breaking change with new build errors might be net positive. IMO opinion it might be interesting to see the customer reaction to the warnings. We should probably add breaking change notes to the RC-1 blog though.

Reply:
> We should probably add breaking change notes to the RC-1 blog though.

Yes, agreed.

Reply:
https://github.com/dotnet/core/issues/8439.